Makefile

```diff
@@ -205,6 +205,9 @@ deploy_cr:
 validate-no-offensive-lang:
 	./hack/validate-no-offensive-lang.sh
 
+metricsdocs:
```

Comment:
What about adding this into `generate-doc` or adding this into `generate-doc`'s dependent? 

`generate-doc` is triggered by `make sanity` and the sanity checks ensure that the generated doc is up-to-date according to the code. Could you please check this part?